src/participant/participant.ts

```diff
@@ -236,22 +243,34 @@ export default class ParticipantController {
     modelInput: ModelInput;
     stream: vscode.ChatResponseStream;
     token: vscode.CancellationToken;
-  }): Promise<void> {
+  }): Promise<{
+    outputLength: number;
+    hasCodeBlock: boolean;
```

Comment:
Should we do codeBlockCount ? Or boolean fine? Not a blocker

Reply:
Fine either way - I went with a bool because that was the type in the design.